smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy

```diff
@@ -380,8 +383,6 @@ def discovery() {
             	log.debug "valid Token"
                 state.Harmonydevices = response.data
                 state.resethub = false
-                getActivityList()
-                poll()
```

Comment:
Those actually update a device that already exist. During discovery we still don't have the devices so no point in trying to find data yet.